ui/pages/nightlies.js

```diff
@@ -209,14 +209,16 @@ export const getLatestNightlies = async (cutoffDate, limit = 1, pipeline) => {
   return Promise.all(
     pipelines.map(async obj => {
       const details = await getPipelineDetails(pipeline, obj.iid);
+      const shiftedDate = dayjs(obj.created_at).add(12, 'hours').toISOString();
```

Comment:
my only suggestion would be that the longer explanation from the commit would be more visible in a comment here, where it can evolve along with the code once e.g. a new pipeline is added to the set of nightlies and then the timeshift would need adjustment (say if we get a lot of pipeline overlap during the night)

Reply:
Good feedback - I'll try to be mindful about these in the future. I added a comment now.